wiki/BanchoBot/it.md

```diff
@@ -1,35 +1,159 @@
 ---
-outdated_translation: true
+tags:
+  - bancho
+  - server bot
+  - commands
+  - announcements
 ---
 
+<!--TODO:
+- add section that lists and explains all the game-related announcements -->
+
 # BanchoBot
 
-![BanchoBot's player card](img/BanchoBot.jpg "BanchoBot's player card")
+*Per altri utilizzi, consulta [Bancho (disambiguazione)](/wiki/Disambiguation/Bancho).*
+
+![La scheda utente di BanchoBot](img/BanchoBot.jpg "La scheda utente di BanchoBot")
+
+**BanchoBot** (a volte indicato come *Bancho*) è un bot di chat online creato per osu! che aiuta le persone nella chat in-game annunciando messaggi relativi al gioco (ad esempio, partite effettuate, tentativi, ecc.) e rispondendo a determinati comandi. È stato programmato da ::{ flag=NZ }:: [Echo](https://osu.ppy.sh/users/431) ed è il fornitore di [Bancho IRC](/wiki/Community/Internet_Relay_Chat) (Internet Relay Chat).
+
+BanchoBot ha anche il proprio [profilo di osu!](https://osu.ppy.sh/users/3) e un [account Twitter](https://twitter.com/banchoboat).
+
+## Commands
+
+*Per una lista di tutti i comandi del client in-game, consulta: [Chat Console](/wiki/Client/Interface/Chat_console)*
+
+BanchoBot può rispondere ai comandi dei giocatori attraverso messaggi specifici inseriti nella chat. Tutti i comandi di BanchoBot iniziano con un punto esclamativo (`!`) seguito dal nome del comando, senza differenza tra maiuscole e minuscole (senza spazio tra di loro). Questi comandi possono essere utilizzati nei canali di chat e tramite messaggi privati con BanchoBot.
+
+Se un utente normale invia un comando in un canale di chat pubblico, gli altri utenti non lo vedranno e la risposta verrà mostrata in un messaggio privato con BanchoBot. Gli utenti possono anche utilizzare il comando del client in-game /bb per aprire automaticamente una scheda con BanchoBot e inviare immediatamente il comando.
+
+Di seguito è riportata una lista di tutti i comandi di BanchoBot:
+
+- [Help](#help)
+- [Roll](#roll)
+- [Stats](#stats)
+- [Where](#where)
+- [FAQ](#faq)
+- [Report](#report)
+
+### Help
+
+```
+!help
+```
+
+`!help` mostra la lista di tutti i comandi di BanchoBot. Di seguito un esempio se si usa questo comando.
```

Comment:
```suggestion
`!help` mostra la lista di tutti i comandi di BanchoBot. Di seguito, un esempio dell'output del comando:
```
This might be a bit different from the original but I think it is way clearer